### ISSUE TITLE
Creating OSM numbers for new Sydney compounds prior to submission for biological evaluation.

### ISSUE THREAD

**Issue by alintheopen** [OpenSourceMalaria/OSM_To_Do_List#296]
A call to the community for some help in adding OSM numbers to a selection of compounds prior to submission.

Details of how to assign OSM numbers here: http://malaria.ourexperiment.org/uri/4cc

Most pressing are:

AEW 240-1  (http://malaria.ourexperiment.org/uri/7e3)
AEW 239-1 (http://malaria.ourexperiment.org/triazolopyrazine_se/10731/Synthesis_of_1phenyl236trifluoromethylpyridin3yl124triazolo43apyrazin5yloxyethan1ol_AEW_2391.html)
AEW 238-1
(http://malaria.ourexperiment.org/triazolopyrazine_se/10736/Synthesis_of_23ethyl124triazolo43apyrazin5yloxy1phenylethan1ol_AEW_2381.html)
AEW 237-1
(http://malaria.ourexperiment.org/triazolopyrazine_se/10784/Synthesis_of_23cyclohexyl124triazolo43apyrazin5yloxy1phenylethan1ol_AEW_2371.html)
AEW 236-1
(http://malaria.ourexperiment.org/triazolopyrazine_se/10794/Synthesis_of_234difluoromethoxyphenyl124triazolo43apyrazin5yloxy1phenylethan1ol_AEW_2361.html)
AEW 231-1
(http://malaria.ourexperiment.org/triazolopyrazine_se/11052/Synthesis_of_R234difluoromethoxyphenyl124triazolo43apyrazin5yloxy1phenylethan1amine_AEW_2311.html)
AEW 230-1
(http://malaria.ourexperiment.org/triazolopyrazine_se/11086/Synthesis_of_1phenyl23phenyl124triazolo43apyrazin5yloxyethan1ol_AEW_2301.html)
AEW 214-1
(http://malaria.ourexperiment.org/triazolopyrazine_se/11373/Synthesis_of_R234fluoromethoxyphenyl124triazolo43apyrazin5yloxyNmethyl1phenylethan1amine_AEW_2141.html)

and then as a secondary task all of the intermediates used to make these compounds.

If you think you can help then comment here or tweet to ask for the OSDD password.

Cheers

Alice

**Comment by drc007** [comment]
http://malaria.ourexperiment.org/osm_procedures/group/Summary%20Page

I don't see any Strings?

**Comment by mattodd** [comment]
Hi @alintheopen  - we’ve a student volunteer (Sebastien Dath: @sebjd ) who is interested in adding the new compounds under evaluation to the wiki and the Google sheet. He has access to Chemdraw so can generate strings. What he needs is the correspondence between structures and OSM numbers - you just assigned these before shipping, right? Could you point him to a place where he can access this info? If you've a Chemdraw to hand could you upload to the Biological Evaluation ELN? Ta.

**Comment by alintheopen** [comment]
coming asap.

**Comment by alintheopen** [comment]
Hi @sebjd the compounds are uploaded with ChemDraw here: http://malaria.ourexperiment.org/biological_data/11082/Samples_sent_for_in_vitro_efficacy_evaluation_against_Plasmodium_falciparum_asexual_blood_stage.html

If you can generate strings I will then add these to the ELN. The OSM numbers below the compounds need to be assigned formally on the ELN too. Many of the precursors (hydrazinyl intermediates, chloro-triazolopyrazines etc.) also require OSM numbers but it would be super if you could do the Syngene compounds first.

Cheers for the help!
Alice

**Comment by sebjd** [comment]
Hi @alintheopen ,  I've managed to generate the SMILES and InChl strings for the attached .cdmx file at the bottom of the linked page, I'm just unsure where you want me to put them now

**Comment by drc007** [comment]
Could you add them to this spreadsheet, https://docs.google.com/spreadsheets/d/1Rvy6OiM291d1GN_cyT6eSw_C3lSuJ1jaR7AJa8hgGsc/edit#gid=510297618

**Comment by sebjd** [comment]
Hi @drc007, I'm assuming I just add new rows to the spreadsheet as they don't seem to already be there? If so, i'm unsure what to put in for the first column "ID" . Sorry for the hassle, still trying to get the hang of it

**Comment by drc007** [comment]
Hi @sebjd, Yes add new rows, and someone else will add the MMV numbers.

**Comment by sebjd** [comment]
Hi, OSM-S-275:283 should hopefully be added in now

**Comment by alintheopen** [comment]
Thanks! Did you want to update the ELN manually too, or are we going to try
to use this spreadsheet to populate the list from now on @drc007?

**Comment by drc007** [comment]
Hi @alintheopen, @mattodd , I think you need someone with a better insight into the overall workflow than I have to decide how this will be done in the future. At the moment we don't have the scripts in place to automate this.

**Comment by alintheopen** [comment]
Thanks @drc007. @sebjd do you want to have a go at assigning following this guide: (http://malaria.ourexperiment.org/osm_logos_and_templ/8913/How_to_Maintain_the_OSM_Compound_Numbering_System.html)?

**Comment by mattodd** [comment]
Indeed it would be amazing if these could be added to the relevant ELN - a
little more work, but valuable. We're up to OSM-S-274

http://malaria.ourexperiment.org/osm_procedures

On 4 June 2015 at 21:18, alintheopen <notifications@github.com> wrote:

>  Thanks @drc007 <https://github.com/drc007>. @sebjd
> <https://github.com/sebjd> do you want to have a go at assigning
> following this guide: (
> http://malaria.ourexperiment.org/osm_logos_and_templ/8913/How_to_Maintain_the_OSM_Compound_Numbering_System.html
> )?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/296#issuecomment-108845840>
> .
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/chemistry/research/todd.html | W
http://opensourcemalaria.org/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.

**Comment by alintheopen** [comment]
Hi @sebjd do you have any time or need any help with the ELN work? No problem either way - just want to follow up on this issue :)

**Comment by sebjd** [comment]
Hi, so sorry about the lack of response, been very busy past few weeks due to exam period, I should be able to help out after this weekend

**Comment by alintheopen** [comment]
No worries at all and good luck with your last exams :) Thanks!

**Comment by sebjd** [comment]
Hi @alintheopen I can help out a bit now, i'm unsure what/where the ELN is however

**Comment by alintheopen** [comment]
Hi @sebjd awesome! Hope the exams went well.
The mighty ELN or electronic laboratory notebook is here:
http://malaria.ourexperiment.org/dashboard
The 'how to' for numbering compounds is here: http://malaria.ourexperiment.org/osm_logos_and_templ/8913/How_to_Maintain_the_OSM_Compound_Numbering_System.html

We'll need to give you login rights so once you've read through the instructions and you're happy, email @mattodd and he will send you the login info. We have *loads* of compounds to add to this database so any help is much appreciated! Let's start with the compound that have been assigned an OSM number in the google sheet (https://docs.google.com/spreadsheets/d/1Rvy6OiM291d1GN_cyT6eSw_C3lSuJ1jaR7AJa8hgGsc/edit#gid=510297618) but don't yet feature on the ELN here (http://malaria.ourexperiment.org/osm_procedures) and here (http://malaria.ourexperiment.org/osm_procedures/group/Compound+List).

So, basically this means starting from OSM-S-275 to OSM-S-283.

Let me know if you need some more help!

Cheers

Alice
